lib/db/memory.js

```diff
@@ -187,7 +203,54 @@ MemoryStore.prototype = {
     deleteByUserId(this.tokens, userId);
     deleteByUserId(this.codes, userId);
     return P.resolve();
-  }
+  },
+  developerRegister: function developerRegister(email) {
```

Comment:
nit: the verb/noun order on these names is inconsistent with the existing methods in this module, e.g. `removeUser` versus `developerRemove`.